docker/Dockerfile

```diff
@@ -447,6 +450,11 @@ COPY ./test_startup.sh /usr/libexec/pyrex/startup.d/
 # Precompile python files for improved startup time
 RUN python3 -m py_compile /usr/libexec/tini/*.py
 
+# Remove all non-root users and groups so that there are no conflicts when the
+# user is added
+RUN getent passwd | cut -f1 -d: | grep -v '^root$' | xargs -L 1 userdel
+RUN getent group | cut -f1 -d: | grep -v '^root$' | xargs -L 1 groupdel
```

Comment:
Random minor tip for future reference, grep -x will match the whole line. Ex. `grep -vx root`